### PR TITLE
fix default WASMER_DIR error; improve error messages on missing runtime

### LINKED ISSUE #105: Defaulting to current dir if WASMER_DIR is not set
This seems to be a result of a bug in #69 ;

- [x] Fix this
- [x] Add file detection to the regression test

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
fixes #105

### CODE REVIEW

File: end-to-end-tests/install.txt [hunk 1]
```diff
@@ -21,15 +21,18 @@ LOCAL COMMANDS:
...
+[INFO] Package "mark/rust-example" is uninstalled.
 No packages found
 [INFO] Installing mark/wapm-override-test@0.1.0
 Global package installed successfully!
 Version 0.1.0
 [INFO] Installing mark/wapm-override-test@0.2.0
+Package installed successfully to wapm_packages!
 Version 0.2.0
-[INFO] Package "wapm-override-test" is uninstalled.
+[INFO] Package "mark/wapm-override-test" is uninstalled.
```
Comment (xmclark) [inline end-to-end-tests/install.txt]: I'm ok with this...are you ok with this?

Comment (xmclark) [reply end-to-end-tests/install.txt]: @MarkMcCaskey

File: src/config.rs [hunk 2]
```diff
@@ -51,15 +51,20 @@ impl Default for Config {
 
 impl Config {
     pub fn get_folder() -> Result<PathBuf, GlobalConfigError> {
-        Ok(env::var(GLOBAL_CONFIG_FOLDER_ENV_VAR)
-            .map(|folder_str| PathBuf::from(folder_str))
-            .unwrap_or({
+        Ok(
+            if let Some(folder_str) = env::var(GLOBAL_CONFIG_FOLDER_ENV_VAR)
+                .ok()
+                .filter(|s| !s.is_empty())
```
Comment (xmclark) [inline src/config.rs]: couldn't find a more clever way to do this. well done.

File: end-to-end-tests/install.txt (hunk 1 above)
Comment (MarkMcCaskey) [reply end-to-end-tests/install.txt]: Yeah, I wish it had been caught before; it's weird that it broke on this branch and not the branch that changed it 🤔

**Pull request merged**